reactapp/src/actions/reviewActions.js

```diff
@@ -12,20 +12,18 @@ export const GET_REVIEW_ID_ERROR = 'GET_REVIEW_ID_ERROR';
 export const ADDING_REVIEW = 'ADDING_REVIEW';
 export const ADDED_REVIEW = 'ADDED_REVIEW';
 export const ADD_REVIEW_ERROR = 'ADD_REVIEW_ERROR';
-// willUpdateReview
-export const WILL_UPDATE_REVIEW = 'WILL_UPDATE_REVIEW';
 // updateReview
 export const UPDATING_REVIEW = 'UPDATING_REVIEW';
 export const UPDATED_REVIEW = 'UPDATED_REVIEW';
 export const UPDATE_REVIEW_ERROR = 'UPDATE_REVIEW_ERROR';
-// willDeleteReview
-export const WILL_DELETE_REVIEW = 'WILL_DELETE_REVIEW';
 // deleteReview
 export const DELETING_REVIEW = 'DELETING_REVIEW';
 export const DELETED_REVIEW = 'DELETED_REVIEW';
 export const DELETE_REVIEW_ERROR = 'DELETE_REVIEW_ERROR';
-// showReviewModal
-export const SHOW_REVIEW_MODAL = 'SHOW_REVIEW_MODAL';
+// likeReview
+export const LIKING_REVIEW = 'LIKING_REVIEW';
```

Comment:
"Error: could not update like data" when trying to like my own review. Should we be able to attempt to review our own review?

Reply:
Fixed!
Review modal had been set up to show own reviews as likable for testing purposes.